net/tsaddr/tsaddr.go

```diff
@@ -213,3 +213,25 @@ func PrefixIs4(p netaddr.IPPrefix) bool { return p.IP().Is4() }
 
 // PrefixIs6 reports whether p is an IPv6 prefix.
 func PrefixIs6(p netaddr.IPPrefix) bool { return p.IP().Is6() }
+
+// ContainsExitRoutes reports whether rr contains both the IPv4 and
+// IPv6 /0 route.
+func ContainsExitRoutes(rr []netaddr.IPPrefix) bool {
+	var v4, v6 bool
+	for _, r := range rr {
+		if r.Bits() != 0 {
+			continue
+		}
+		if r == AllIPv4 {
+			v4 = true
+		} else if r == AllIPv6 {
+			v6 = true
+		}
+	}
+	return v4 && v6
+}
+
+var (
+	AllIPv4 = netaddr.MustParseIPPrefix("0.0.0.0/0")
```

Comment:
Make these funcs? So that they can't be modified

Reply:
done